etc/openqa/openqa.ini

```diff
@@ -325,3 +325,8 @@ concurrent = 0
 # home =
 # username = openqa-user
 # ssh_key_file = ~/.ssh/id_rsa
+
+[test_preset bar]
+title = Some preset
+distri = does-not-exist
+casedir = http://foo.git
```

Comment:
Those should be commented by default. And can you add some explanation comments here? I wouldn't know why I would want to set those settings. And please use the actual default values

Reply:
This should have been added to the config file we use for tests. I didn't intend to touch the normal config template at all for now.